### ISSUE TITLE
fatal error on testnet

### ISSUE THREAD

**Issue by prudnitskiy** [digibyte/digibyte#102]
Trying to invoke digibyted with a testnet setting cause fatal error:

```digibyted[2667]: setKeyPool.size() = 100
digibyted[2667]: mapWallet.size() = 0
digibyted[2667]: mapAddressBook.size() = 1
digibyted[2667]: ERROR: ReadBlockFromDisk: Errors in block header at CBlockDiskPos(nFile=0, nPos=236)
digibyted[2667]: *** Failed to read block
digibyted[2667]: Error: Error: A fatal internal error occurred, see debug.log for details
digibyted[2667]: Error: Error: A fatal internal error occurred, see debug.log for details
digibyted[2667]: Failed to connect best blockFailed to open mempool file from disk. Continuing anyway.
```

Daemon will work but will not trying to sync blockhain.

This error can be reproduced both on 6.14.1 and 6.14.2

Error can be reproduced on VM or real hardware. 

Cleaning up data dir will not help.

OS: Debian 8.9 x64

**Comment by tjazak** [comment]
Any solution to this yet? I need to be able to use regtest to debug an application.

**Comment by jdabs** [comment]
I believe there's ongoing development effort for this. I don't have an update yet.

**Comment by skawhdgjs** [comment]
change nounce value when create genesisblock    /src/chainparams.cpp Regtest

**Comment by digicontributer** [comment]
Fixed in 6.16.2
